types/react/index.d.ts

```diff
@@ -2551,7 +2553,7 @@ declare namespace React {
         noModule?: boolean | undefined;
         referrerPolicy?: HTMLAttributeReferrerPolicy | undefined;
         src?: string | undefined;
-        type?: string | undefined;
+        type?: LiteralUnion<'module' | 'importmap'>;
```

Comment:
```suggestion
        type?: LiteralUnion<'module' | 'importmap'> | undefined;
```

Please don't remove the `| undefined`s; they are needed for `exactOptionalPropertyTypes`.